src/main/java/uk/gov/hmcts/probate/services/pin/controllers/PinController.java

```diff
@@ -11,15 +16,37 @@
 public class PinController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PinController.class);
+    
     @Autowired
     private PinService pinService;
 
-    @RequestMapping(path = "/pin/{phoneNumber}", method = RequestMethod.GET)
-    public String invite(@PathVariable String phoneNumber,
-                         @RequestHeader("Session-Id") String sessionId) throws NotificationClientException {
-
-        LOGGER.info("Processing session id " + sessionId);
+    @RequestMapping(path = "/pin", method = RequestMethod.GET)
+    public ResponseEntity<String> invite(@RequestParam String phoneNumber,
+                         @RequestHeader("Session-Id") String sessionId) throws NotificationClientException, UnsupportedEncodingException {
+ 	
+		if ( null == sessionId ) {
+			LOGGER.error("Session-Id request header not found");
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
+		}
+    	LOGGER.info("Processing session id " + sessionId);
+    	
+		phoneNumber = URLDecoder.decode(phoneNumber, java.nio.charset.StandardCharsets.UTF_8.toString());
+		phoneNumber = phoneNumber.replaceAll("[ \\(\\)\\[\\]-]", "");			
+		
+		if ( !phoneNumber.matches("(\\+)*[0-9]+") ) {
```

Comment:
Please remove all unnecessary spaces - keep it consistent throughout the code base.